src/spikeinterface/exporters/to_phy.py

```diff
@@ -167,7 +167,7 @@ def export_to_phy(
         f.write(f"dtype = '{dtype_str}'\n")
         f.write(f"offset = 0\n")
         f.write(f"sample_rate = {fs}\n")
-        f.write(f"hp_filtered = {sorting_analyzer.recording.is_filtered()}")
+        f.write(f"hp_filtered = {sorting_analyzer.is_filtered()}")
```

Comment:
I know we need this for compatibility, but I'm pretty sure Phy has stopped checking the filter status ages ago. KS just always wrote this so we have to keep it around. The cost of tech debt...